extra/resources/ClusterMon.in

```diff
@@ -170,7 +170,7 @@ do
     case "$OPTION" in
     V|n|r|c|d);;
     i)  ocf_log warn "You should not have specified the -i option, since OCF_RESKEY_update is set already!";;
-    h)  ocf_log warn "You should not have specified the -h option, since OCF_RESKEY_htmlfile is set already!";;
+    h)  ocf_log warn "You should not have specified the --output-as=html --output-to= option, since OCF_RESKEY_htmlfile is set already!";;
```

Comment:
This is a little different. This function checks what the user passed as the extra_options parameter, so "h)" in this case is what the user passed, meaning the original text was correct here.

However that does raise the point that we should check that the user did not pass --output-as or --output-to, but I think the current design can only handle single-letter options. I think maybe we could check that with a separate case statement like

    case "$OCF_RESKEY_extra_options" in
        *--output-as*) ...
        *--output-to*) ...
    esac